index.html

```diff
@@ -17,7 +17,7 @@
     <header>
       <nav class="nav">
         <h1 class="my-logo">My logo</h1>
```

Comment:
Kindly make use of a tag to wrap the h1 so that it can be accessed by other devices such as tab and screen readers.